### DISCUSSION TITLE
cannot flash the AIdeck and how can i reset the code have flashed in

### DISCUSSION THREAD

**Discussion by softword-tt** [bitcraze/discussions#1175] [Q&A]
hiiiiiiii~ i write a custom classification model and i try to test with Crazyflie and aideck. And i wrote a loop cpxprint code ( while1 print) yesterday (T_T), after that when i connect with cfclient, (crazyflie+aideck) took much longer time than only crazyflie. And when i try to flash a new code, the terminal is stuck in "Reset to bootloader mode", when I take the aideck off, it can exit successfully. The same case with the helloworld demo. 
<img width="974" alt="Snipaste_2024-02-07_21-55-28" src="https://github.com/bitcraze/discussions/assets/63800525/c8c8bb8d-7faa-480e-b175-4f7939f9385c">
I also use cfclient>bootloader to flash the firmware, and I usually use the code boot way, since the connection with crazyflie+aideck takes long time after the bug.  Code boot firmware is ok and the console continuously and fastly prints the stupid code I have written. So for this (Reset to bootloader mode) problem I guess its because of my print code, so do you have some other ideas or how can I clear the code and reset the aideck? thx a lot!!

**Comment by knmcguire** [comment]
hi! In see the discussion has been closed, does that mean that you solved the issue?
